scripts/run-docker.sh

```diff
@@ -15,14 +17,17 @@ docker start $CONTAINER_NAME > /dev/null 2> /dev/null || {
 	       --detach \
 	       --env HOME=$HOME \
 	       --name $CONTAINER_NAME \
-	       --volume $PWD:$HOME/termux-packages \
+	       --volume $REPOROOT:$HOME/termux-packages \
 	       --tty \
 	       $IMAGE_NAME
-
-	echo "Changed builder uid/gid..."
-	docker exec $CONTAINER_NAME chown -R `id -u` /data >& /dev/null
-	docker exec $CONTAINER_NAME usermod -u `id -u` builder >& /dev/null
-	docker exec $CONTAINER_NAME groupmod -g `id -g` builder >& /dev/null
+	if [ $(id -u) -ne 1000 ]
```

Comment:
Good catch, you are right! I just took @hacknox patches and made my setup fly.. So you should probably open a new PR for that...